types/validation.go

```diff
@@ -111,6 +123,12 @@ func verifyCommitLightInternal(
 			votingPowerNeeded, ignore, count, countAllSignatures, true)
 	}
 
+	// attempt to verify aggregated commit
```

Comment:
Since you're modifying the "VerifyCommitLight..." functions, please make sure you run an `e2e` test with several validators and at least one light client node.

IICR, `ci.toml` already has some light client nodes, so running `ci.toml` should be enough.

Reply:
> IIRC, ci.toml already has some light client nodes, so running ci.toml should be enough.

That's what I'm using for testing.